dev-client/src/components/map/SiteMap.tsx

```diff
@@ -64,10 +130,39 @@ const SiteMap = memo(({sites, center}: SiteMapProps): JSX.Element => {
       }}
       onLongPress={onLongPress}>
       <Camera ref={camera} centerCoordinate={[0, 0]} />
+      <Mapbox.Images>
+        <Mapbox.Image name="sitePin">
+          <MaterialIconButton name="location-on" />
+        </Mapbox.Image>
```

Comment:
suggestion (nit?): I think this should be just an `Icon` rather than an `IconButton`. As far as I can tell, the child component is just used to render the image.

Related, I made an issue for more generic `Icon` and `IconButton` utility components to simplify our lives #164.